<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AttDialogAdd</class>
 <widget class="QDialog" name="AttDialogAdd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>355</width>
    <height>201</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>160</y>
     <width>181</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="nameLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>67</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameLEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>181</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="amountLabel">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>0</y>
     <width>67</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Amount</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="nullChBox">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>10</y>
     <width>81</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Optional</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="amoutSpinBox">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>20</y>
     <width>52</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimum">
    <number>2</number>
   </property>
   <property name="maximum">
    <number>10</number>
   </property>
  </widget>
  <widget class="QRadioButton" name="simpleButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>110</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Simple</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">typeButtonGroup</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="compoundButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>110</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Compound</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">typeButtonGroup</string>
   </attribute>
  </widget>
  <widget class="QCheckBox" name="multivaluedCBox">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>30</y>
     <width>111</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Multivalued</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>nameLEdit</tabstop>
  <tabstop>simpleButton</tabstop>
  <tabstop>compoundButton</tabstop>
  <tabstop>multivaluedCBox</tabstop>
  <tabstop>amoutSpinBox</tabstop>
  <tabstop>nullChBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AttDialogAdd</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AttDialogAdd</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="typeButtonGroup"/>
 </buttongroups>
</ui>
